Documentation/security/policy/language.rst

```diff
@@ -719,11 +721,15 @@ which is defined as follows:
 
         // PortProtocol specifies an L4 port with an optional transport protocol
         type PortProtocol struct {
-                // Port is an L4 port number. For now the string will be strictly
-                // parsed as a single uint16. In the future, this field may support
-                // ranges in the form "1024-2048"
+                // Port can be an L4 port number, or a name in the form of "http"
+                // or "http-8080".
                 Port string `json:"port"`
 
+                // EndPort can only be an L4 port number.
```

Comment:
Interesting corner case - is it OK for a user to specify something like this?
```
port: "http"
endPort: "8080"
```

Reply:
No. I will be expanding this comment.